hs_core/views/__init__.py

```diff
@@ -2451,7 +2452,7 @@ def dispatch(self, *args, **kwargs):
         return super(MyGroupsView, self).dispatch(*args, **kwargs)
 
     def get_context_data(self, **kwargs):
-        u = User.objects.get(pk=self.request.user.id)
+        u = User.objects.select_related("uaccess").get(pk=self.request.user.id)
```

Comment:
if we `select_related` here, does this impact line 2473 where we return `u`? I don't think so, but just checking...

Reply:
Yes `uaccess` is accessible from `u` without hitting the db again.